### DISCUSSION TITLE
Updating postgres extension

### DISCUSSION THREAD

**Discussion by wadclapp** [supabase/supabase#4193] [Questions] [answered]
Trying to use the postGIS function `ST_ReducePrecision` function and I get the error:
`Precision reduction requires GEOS-3.9 or higher`

`SELECT PostGIS_Full_Version()`
`"POSTGIS=\"3.1.2 cbe925d\" [EXTENSION] PGSQL=\"130\" GEOS=\"3.8.0-CAPI-1.13.1 \" PROJ=\"6.3.1\" LIBXML=\"2.9.10\" LIBJSON=\"0.13.1\""`

`ALTER EXTENSION postgis UPDATE to '3.1.4'` (In the hope it will update GEOS?) gives the error:
`extension "postgis" has no update path from version "3.1.2" to version "3.1.4"`

Does the Supabase team need to make this version available in order for me to upgrade, or is there something I can do?

**Comment by soedirgo** [comment, marked as answer]
Yes, this needs to be updated on our side, which we're in the middle of doing:
- https://github.com/supabase/postgres/pull/98
- https://github.com/supabase/postgres/pull/98/commits/d8fae0f7593c193a8f701cfbdd48058b1c221f52

I don't think extensions are updated for older projects though, so you might need to [migrate to a new project](https://supabase.com/docs/guides/database#migrating-between-projects) for this.

**Reply by wadclapp** [reply]
Thanks for the quick response @soedirgo, I'll give it a try when it's available
